MUXSDKStats/Podfile

```diff
@@ -5,20 +5,23 @@ target 'MUXSDKStats' do
   # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
   # use_frameworks!
   # Pods for MUXSDKStats
-  pod 'Mux-Stats-Core', '~>3.6'
+  #  pod 'Mux-Stats-Core', '~>3.6'
+  pod 'Mux-Stats-Core', :git => 'https://github.com/muxinc/stats-sdk-objc.git', :branch => 'release/3.8.0'
```

Comment:
I think we need to update this podfile and the podfiles for all the demo apps to point to the right version of Mux Core

Reply:
Updated